client/ext/bigquery/bigquery-module.js

```diff
@@ -0,0 +1,14 @@
+goog.provide('p3rf.perfkit.explorer.ext.bigquery.module');
+
+goog.require('p3rf.perfkit.explorer.ext.bigquery.BigqueryConfigModel');
+goog.require('p3rf.perfkit.explorer.ext.bigquery.BigqueryConfigService');
+goog.require('p3rf.perfkit.explorer.ext.bigquery.CurrentTimestampOptimizer');
+
+
+goog.scope(function() {
+  const bigquery = p3rf.perfkit.explorer.ext.bigquery;
+  
+  bigquery.module = angular.module('pkx.bigquery', []);
```

Comment:
I haven't seen the name 'pkx' used before in code - client/app.js uses 'explorer'. Was this intentional? Not sure if it matters, but would be nice to keep this consistent.